wavefront/server/modules/voice_agents_module/voice_agents_module/services/exotel_service.py

```
@@ -0,0 +1,106 @@
+                    logger.info(
+                        f'Exotel call created successfully. Call SID: {call_sid}'
+                    )
+
+                    return {
+                        'call_sid': call_sid,
+                        'status': status,
+                        'to_number': to_number,
+                        'from_number': from_number,
+                    }
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)